packages/vitest/src/node/reporters/utils.ts

```diff
@@ -1,12 +1,13 @@
 import type { ViteNodeRunner } from 'vite-node/client'
+import { normalize } from 'pathe'
 import type { Reporter } from '../../types'
 import { BenchmarkReportsMap, ReportersMap } from './index'
 import type { BenchmarkBuiltinReporters, BuiltinReporters } from './index'
 
 async function loadCustomReporterModule<C extends Reporter>(path: string, runner: ViteNodeRunner): Promise<new () => C> {
   let customReporterModule: { default: new () => C }
   try {
-    customReporterModule = await runner.executeId(path)
+    customReporterModule = await runner.executeId(normalize(`/@fs/${path}`))
```

Comment:
```suggestion
    customReporterModule = await runner.executeFile(path)
```

I think this would do.